.github/workflows/publish.yaml

```diff
@@ -43,9 +44,18 @@ jobs:
           # Make sure it is installable.
           sudo snap install --dangerous --classic ${{ steps.build-snapcraft.outputs.snap }}
 
-      - if: steps.decisions.outputs.PUBLISH == 'true' && steps.decisions.outputs.PUBLISH_BRANCH != null
+      - if: steps.decisions.outputs.PUBLISH == 'legacy' && steps.decisions.outputs.PUBLISH_BRANCH != null
         uses: snapcore/action-publish@v1
         with:
           store_login: ${{ secrets.STORE_LOGIN }}
           snap: ${{ steps.build-snapcraft.outputs.snap }}
           release: ${{ steps.decisions.outputs.PUBLISH_BRANCH }}
+
+      - if: steps.decisions.outputs.PUBLISH == 'env' && steps.decisions.outputs.PUBLISH_BRANCH != null
+        # Use this until snapcore/action-publish#27 it is merged.
+        uses: sergiusens/action-publish
```

Comment:
Will this eventually be merged into or replace snapcore/action-publish?

Reply:
it should be merged into snapcore/action-publish